static/js/inputHandler.js

```diff
@@ -22,6 +23,11 @@ inputHandler = () => {
       toCompute = input.name;
     }
     else input.value = element.value;
+
+    if(input.name == 'length' || input.name == 'r0' || input.name == 'ra'
+     || input.name == 'x'|| input.name == 'force') {
+        input.unit = unit.value
+    }
```

Comment:
I'm not sure, but would if( input.name !== 'voltage' && input.name !== 'awg') be more efficient?

Reply:
You are right. Simplified it.